cmsdb/processes/higgs.py

```diff
@@ -70,52 +90,69 @@
     name="zh",
     id=13100,
     label="ZH",
-    xsecs={13: Number(0.1)},  # TODO
+    xsecs={
+        13: Number(0.1),  # TODO
+        13.6: Number(0.9439, {
+            "scale": (0.037j, 0.032j),
+            "pdf": 0.016j,
```

Comment:
```suggestion
        13.6: Number(0.9439, {  # value for mH=125 GeV
            "scale": (0.037j, 0.032j),
            "pdf": 0.016j,
```

I would add a comment to say that you used the values for a Higgs mass 125 GeV (for bookkeeping)